docs/tutorial.md

```diff
@@ -0,0 +1,506 @@
+---
+id: tutorial
+title: Tutorial
+---
+
+This article is meant to accompany the "Getting Started" video. Follow along with the video to gain deeper insights into getting started with Demisto.
+
+[![Getting Started](https://img.youtube.com/vi/bDntS6biazI/0.jpg)](https://youtu.be/bDntS6biazI)
+
+## Welcome
+
+>*First and foremost, I want to welcome you to the Demisto team. We are excited to have you and we look forward to working with you. Please note that this tutorial assumes that you have a working instance of Demisto.*
+
+The code we will be writing will be available in segments as we go along, as well as in it's entirety at the end.
+
+## Navigating to BYOI
+
+![byoi](https://user-images.githubusercontent.com/42912128/50827466-30922280-1347-11e9-9eb1-cac42ed62e17.png)
```

Comment:
Where do these images link come from? For now we can leave the links as I believe we're going to refactor this tutorial soon, but if we keep them we should move these to the content-docs repo.

Reply:
I downloaded the images. They should really be local. We don't have a guarantee how long they will be stored.